### ISSUE TITLE
macOS build fails because it tries to build Debian package

### ISSUE THREAD

**Issue by koenvervloesem** [rhasspy/rhasspy-hermes#6]
The `make dist` target creates a Debian package, also on macOS, where it fails. See https://github.com/rhasspy/rhasspy-hermes/runs/376446821

Not sure how to handle this, is there a situation where it makes sense to build a Debian package on macOS? Or should we only create the Debian package on the Ubuntu target in the GitHub Action workflow?

**Comment by maxbachmann** [comment]
@koenvervloesem on 3.8 it fails both for ubuntu and mac os because [pyinstaller](https://github.com/pyinstaller/pyinstaller) does not support python 3.8 yet [issue](https://github.com/pyinstaller/pyinstaller/issues/4311)

the other errors on mac os are not related to the code itself, but to the commandset the Makefile is using
```
cat debian/DEBIAN/control | version=0.1.5 architecture= envsubst > "debian/rhasspy-hermes_0.1.5_/DEBIAN/control"
```
while on linux envsubst is a build in it is not on mac os (would require gettext)

**Comment by koenvervloesem** [comment]
Thanks, @maxbachmann, the Python 3.8 issue is another one indeed, see https://github.com/rhasspy/rhasspy-microphone-cli-hermes/issues/5 for the issue I created this morning and a possible fix.

I could install gettext in the macOS runner with brew, and it probably needs fakeroot and dpkg too, and see how it goes.

**Comment by maxbachmann** [comment]
I am not sure about mac os but at least the linux images for the github ci all have passwordless root I guess it is probably the same for mac os

**Comment by koenvervloesem** [comment]
Fixed: https://github.com/rhasspy/rhasspy-hermes/runs/377403293

The Debian package is built now on macOS too by installing the required dependencies with brew.

I temporarily disabled the Python 3.8 runner to be able to test this without cluttering the test results, I'll fix that issue later and re-enable Python 3.8 then.
